library/lvm_gensym.py

```diff
@@ -55,22 +56,28 @@
     type: str
 '''
 
-def get_os_name():    
+
+def get_os_name():
+    """Searches the hosts /etc/os-release file and outputs the name in the ID column"""
     for line in open('/etc/os-release').readlines():
         if not line.find('ID='):
             os_name = line[3:]
             break
-        
+
     os_name = os_name.replace('\n', '').replace('"', '')
-    return os_name 
+    return os_name
 
 def name_is_unique(name, used_names):
+    """Given a string input and a list of strings, function returns a boolean value based on whether
+       that item exists in the used_names lists"""
```

Comment:
One comment on docstrings that applies to most of yours:

> The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".

(From https://www.python.org/dev/peps/pep-0257/#one-line-docstrings)